src/unitxt/catalog.py

```diff
@@ -97,16 +100,23 @@ def __contains__(self, artifact_identifier: str):
 def verify_legal_catalog_name(name):
     assert re.match(
         r"^[\w" + COLLECTION_SEPARATOR + "]+$", name
-    ), 'Catalog name should be alphanumeric, ":" should specify dirs (instead of "/").'
+    ), f'Catalog name should be alphanumeric, ":" should specify dirs (instead of "/"). Name: {name}'
```

Comment:
I think this error is phrased in a confusing way, but it is not a blocker

Reply:
Improved